### PR TITLE
feat(xml_query_fix): Refactor Xmlfinder Aggregation Query

### PR DESCRIPTION
**Pull request by tholulomo**
Fixing aggregation pipeline query to return a total number of xml available. This is required for pagination

### CODE REVIEW

**Review by aswallace — COMMENTED** [review]
Added the UI changes: 
- `isNewCuration` parameter to every place an XML is linked, and also in the XmlLoader.vue page
- Filters in the admin "Manage Curation" page
- Refactored unit tests

Still seeing some errors when I search by approval status and curating user

File: resfulservice/src/pipelines/curation-pipeline.js [hunk 1]
```diff
@@ -54,9 +54,9 @@ exports.curationSearchQuery = async (input) => {
...
     },
     { $match: filter },
-    { $skip: skip },
-    { $limit: pageSize }
+    { $group: { _id: null, count: { $sum: 1 }, xmlData: { $push: '$$ROOT' } } },
+    { $project: { _id: 0, count: 1, xmlData: { $slice: ['$xmlData', skip, pageSize] } } }
   ]);
-
-  return { xmlData, count: xmlData.length };
+  const { xmlData, count } = data[0];
```
[the remark this answers is not in the document]
Comment (tholulomo) [reply resfulservice/src/pipelines/curation-pipeline.js:60]: I see, your xml collection shouldn't be empty, but if it is then you will see this issue.

**Review by aswallace — APPROVED** [review]
Reviewed the xsd schema part for the admin portal, works for me locally

File: resfulservice/src/pipelines/curation-pipeline.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (aswallace) [reply resfulservice/src/pipelines/curation-pipeline.js:60]: I see. I do have XMLs loaded, but maybe they're in a different collection and it's just this specific one that's empty? Should I resolve this comment, or does it still need to be addressed?

**Pull request merged**